starfinder/db/models.py

```diff
@@ -128,30 +128,59 @@ def __repr__(self):
 # -----------------
 
 class Equipment(db_engine.Model, ModelBase, HasId):
+    id = db_engine.Column(db_engine.Integer(), primary_key=True)
```

Comment:
this is a case where we'll actually want to use a `GUID`, since this will represent a user's equipment

General rule: 
user input = unique identifier
core data = incremented identifier